notebooks/render_fn.clj

```diff
@@ -0,0 +1,14 @@
+(require '[nextjournal.clerk :as clerk])
+
+#_(clerk/clear-cache!)
```

Comment:
@mk This should not be necessary anymore after you implement content-hashing on atom contents